theme/components/stickytoolbar.scss

```diff
@@ -5,14 +5,12 @@
 	.ck-toolbar.ck-toolbar_sticky {
 		@include ck-drop-shadow();
 
-		position: fixed;
 		top: 0;
```

Comment:
Shouldn't it be a part of Wireframe? I'm confused about `top, left, right, bottom` and `transform: translate()`. On one hand, they're very basic definitions of what the component should look like. On the other hand, they could be customized by other themes.